test/helpers/gas-usage.ts

```diff
@@ -36,19 +36,19 @@ const MAX_GAS_PER_GROUP: any = {
   [GasGroup.REMOVE_OPERATOR]: 62000,
   [GasGroup.REMOVE_OPERATOR_WITH_WITHDRAW]: 62000,
 
-  [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 187000,
-  [GasGroup.REGISTER_VALIDATOR_NEW_STATE]: 204000,
-  [GasGroup.REGISTER_VALIDATOR_NEW_STATE_WITHOUT_DEPOSIT]: 166000,
+  [GasGroup.REGISTER_VALIDATOR_EXISTING_POD]: 185300,
```

Comment:
@lior-blox in the path of downsizing gas usage, shot me a smile -:)

Reply:
:)